### PR TITLE
add ~ and ~~ operators😄😁

### PR DESCRIPTION
**Pull request by Nunchakus888**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1351,6 +1352,31 @@ View the FireFox source, `toFixed` method is to convert the value of the convers
 
 * [**20.1.3.3** Number.prototype.toFixed (`fractionDigits`)](https://www.ecma-international.org/ecma-262//#sec-number.prototype.tofixed)
 
+
+### 💡 Explanation:
```
Comment (denysdovhan) [inline README.md]: Something wrong 😕 Why explanation comes before the example?

File: README.md [hunk 2]
```diff
@@ -1351,6 +1352,31 @@ View the FireFox source, `toFixed` method is to convert the value of the convers
...
+
+## `~` and `~~` operator
```
Comment (denysdovhan) [inline README.md]: Add an empty line between paragraphs. Every sentence should start with capitalized word.

File: README.md [hunk 3]
```diff
@@ -1351,6 +1352,31 @@ View the FireFox source, `toFixed` method is to convert the value of the convers
...
+single tide ~ is usual usage as :`~n = -(n+1)`
+
+```
+~false === -1
+~true === -1
+~{} === -1
+~122 === -123
+~-124 === 123
+```
```
Comment (denysdovhan) [inline README.md]: Here again. Spaces between.

Comment (Nunchakus888) [reply README.md]: OK, I has fixed it🙄️

File: README.md [hunk 4]
```diff
@@ -1351,6 +1352,34 @@ View the FireFox source, `toFixed` method is to convert the value of the convers
...
+~~false === 0
+~~0.32 === 0
+~~NaN === 0
+~~-123.321 === -123
+~~(10/3) === 3
+```
+
+### 💡 Explanation:
+
+
```
Comment (denysdovhan) [inline README.md]: Don't forget about explanation.

Comment (Nunchakus888) [reply README.md]: OK🙄️

**Pull request closed without merging**